models/issues/issue_project.go

```
@@ -13,28 +13,22 @@ import (
 )
 
 // LoadProject load the project the issue was assigned to
-func (issue *Issue) LoadProject(ctx context.Context) (err error) {
-	if issue.Project == nil {
-		var p project_model.Project
-		has, err := db.GetEngine(ctx).Table("project").
+func (issue *Issue) LoadProjects(ctx context.Context) (err error) {
+	if len(issue.Projects) == 0 {
+		err = db.GetEngine(ctx).Table("project").
 			Join("INNER", "project_issue", "project.id=project_issue.project_id").
-			Where("project_issue.issue_id = ?", issue.ID).Get(&p)
-		if err != nil {
-			return err
-		} else if has {
-			issue.Project = &p
-		}
+			Where("project_issue.issue_id = ?", issue.ID).Find(&issue.Projects)
 	}
 	return err
 }
 
-func (issue *Issue) projectID(ctx context.Context) int64 {
-	var ip project_model.ProjectIssue
-	has, err := db.GetEngine(ctx).Where("issue_id=?", issue.ID).Get(&ip)
-	if err != nil || !has {
-		return 0
+func (issue *Issue) projectIDs(ctx context.Context) []int64 {
+	var ids []int64
+	if err := db.GetEngine(ctx).Table("project_issue").Where("issue_id=?", issue.ID).Select("project_id").Find(&ids); err != nil {
```

It's not safe to ignore the err, in case the SQL would be wrong.